app/src/lib/fetch.ts

```diff
@@ -1,4 +1,4 @@
-import { getCurrentUser } from "../localStorageManager";
+import { getBearerToken } from "./getBearerToken";
```

Comment:
This is reproducing some stuff from the apiService - again, I think we need some discussion to clarify where we're heading with doing all api access in a consistent way. Also, apiService is using axios rather than fetch, and all the tests for that are using axios MockAdapter, rather than msw. It would be good to have all that unified.

Reply:
created ticket